src/client/components/ProfileCareerResult/ProfileCareerResult.styles.css

```diff
@@ -0,0 +1,51 @@
+.robo_text {
+  margin: auto 2%;
+  font-size: 1.3em;
+}
+
+.profile_career_header {
+  display: flex;
+  flex-direction: row;
+  padding: 3%;
+}
+
+.profile_career_body {
+  margin: 1%;
+  padding: 1%;
+  background-color: #e9f4f4;
+  padding: 2rem;
+}
+
+.profile_career_details {
+  margin: 10% auto 12%;
+  background-color: #e9f4f4;
+  font-size: 1rem;
+}
+
+.profile_career_title {
+  font-family: 'Roboto Mono', monospace;
+  font-size: 42px;
+  margin-bottom: 64px;
+  color: black;
+  line-height: 55px;
+}
+
+.third_word {
+  text-indent: 3rem;
+}
+.frame {
+  position: absolute;
```

Comment:
I can see that you are trying to position this component bottom left using absolute, but you are not the one who has to do that, the person who is creating the container should do that. So just use position relative instead :)